src/components/marquee-wizard/marquee-wizard.component.ts

```diff
@@ -102,37 +92,14 @@ export class MarqueeWizardComponent extends WizardComponent implements OnDestroy
         // get the current step
         const step = this.getCurrentStep() as MarqueeWizardStepComponent;
 
-        // call the original finish function
-        return super.finish().then(() => {
-            // if the step is valid indicate that it is now complete
-            if (step.valid) {
-                step.setCompleted(true);
-            } else {
-                this.stepError.next(this.step);
-            }
-        });
-    }
-
-    /**
-     * If a step in the wizard becomes invalid, all steps sequentially after
-     * it, should become unvisited and incomplete
-     */
-    validChange(state: MarqueeWizardValidEvent): void {
+        await super.finish();
 
-        const steps = this.steps.toArray();
-        const current = steps.findIndex(step => step === state.step);
-        const affected = steps.slice(current);
-
-        affected.forEach(step => {
-
-            // the step should no longer be completed
-            step.completed = false;
-
-            // if the step is not the current step then also mark it as unvisited
-            if (this.resetVisitedOnValidationError && step !== state.step) {
```

Comment:
Looks like you've forced your changes on top of the more recent changes instead of doing a merge. Please revert and re-do this.

Reply:
Apologies I should have seen that in the changes.